src/core/public/public.api.md

```diff
@@ -160,7 +160,8 @@ export interface ApplicationStart {
         deepLinkId?: string;
     }): string;
     navigateToApp(appId: string, options?: NavigateToAppOptions): Promise<void>;
-    navigateToUrl(url: string): Promise<void>;
+    navigateToUrl(url: string, options?: NavigateToUrlOptions): Promise<void>;
+    navigateToUrlSkipUnload(url: string): void;
```

Comment:
Navigation method for security to hook into on session timeouts when a user has to log in again.